jwst/extract_1d/extract_1d_step.py

```diff
@@ -53,6 +53,7 @@ class Extract1dStep(Step):
     soss_width = float(default=40.)  # aperture width used to extract the 1D spectrum from the de-contaminated trace.
     soss_bad_pix = option("model", "masking", default="masking")  # method used to handle bad pixels
     soss_modelname = output_file(default = None)  # Filename for optional model output of traces and pixel weights
+    soss_order_3 = boolean(default=False)
```

Comment:
Is this for testing purposes or are we giving the user the ability to not extract order 3?